utoipa-gen/src/path/response.rs

```diff
@@ -111,12 +100,11 @@ impl ToTokens for Response<'_> {
             utoipa::openapi::ResponseBuilder::new().description(#description)
         });
 
-        if let Some(ref body_type) = self.response_type {
-            let body_ty = &body_type.ty;
+        if let Some(response_type) = &self.response_type {
+            let property = Property::new(response_type.clone());
```

Comment:
This clone could probably be removed if the type was reference or Cow. :) See above :point_up: